ios/CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # CHANGELOG
 
+## 2017-08-18 — LinkKit 1.0.8
+### Added
+- Support more than two answer options on MFA view
```

Comment:
Might be nice to specify the mfa type: `Add support for three or more MFA selection options`